src/theme/index.tsx

```diff
@@ -67,6 +67,11 @@ const fonts = {
   code: 'courier, courier new, serif',
 }
 
+const fontWeight = {
+  book: 485,
+  medium: 535,
```

Comment:
it seems like we never use these outside of the NFT codebase, most places use the hardcoded 485 or 535 values